.rubocop.yml

```diff
@@ -5,6 +5,9 @@ AllCops:
     - "Rakefile"
     - "app.rb"
     - "main.rb"
+    - 'nameable.rb'
+    - 'person.rb'
+    - 'student.rb'
     - "node_modules/**/*"
 
```

Comment:
- [ ] @harshi0102 and @Daniel-Kigozi great work implementing the changes requested, however, you have still some modifications in your robocop file, kindly ensure this file matches the one given in the [ruby linter configuration](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)